### PR TITLE
Imageio v1.3

### PR DESCRIPTION
**Pull request by almarklein**
New release for imageio. Also changed that the freeimage lib is now packed into the library, so that it won't have to be auto-downloaded on first use.

Any chance that CA can build this conda package? I'd love to say that ppl can do `conda install imageio` to install it.

### CODE REVIEW

**Comment by almarklein** [comment]
cc @ccordoba12 and @blink1073

So maybe we should *not* include the freeimage lib in the package, but have a separate conda package for freeimage and depend on that?

**Comment by blink1073** [comment]
For conda purposes, I think it makes sense to have libfreeimage be a separate dependency for imageio.

https://github.com/conda/conda-recipes/tree/master/freeimage

**Comment by ccordoba12** [comment]
I agree with @blink1073, it should managed as a dependency of `imageio`

File: imageio/bld.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"%PYTHON%" setup.py install
+"%PYTHON%" setup.py build_with_fi install
```
Comment (ccordoba12) [inline imageio/bld.bat]: What does this `build_with_fi` flag mean?

Comment (almarklein) [inline imageio/bld.bat]: It means that the freeimage lib is downloaded and included in the package.

**Comment by almarklein** [comment]
Cool, in that case hold of the merging until we change it back, And we need to make sure that imageio can find freeimage where its installed in the conda env.

File: imageio/meta.yaml [hunk 2]
```diff
@@ -1,26 +1,27 @@
 package:
   name: imageio
-  version: 1.2
+  version: 1.3
 
 source:
-  fn: imageio-1.2-linux32.zip  [linux32]
-  url: https://pypi.python.org/packages/source/i/imageio/imageio-1.2-linux32.zip  [linux32]
+  fn: imageio-1.3-linux32.zip  [linux32]
+  url: https://pypi.python.org/packages/source/i/imageio/imageio-1.3-linux32.zip  [linux32]
```
Comment (ccordoba12) [inline imageio/meta.yaml]: Why do you need a *source* file per platform and arch? i don't get it ;-)

Comment (almarklein) [inline imageio/meta.yaml]: Oh wait. Now I'm confused. These per-platform source files include the freeimage lib .... it should be enough to *either* to this or use the `build_with_fi` flag.

Or none of the above, really, if anaconda has freeimage :P

Comment (ccordoba12) [inline imageio/meta.yaml]: Yep, none of the above if Anaconda comes with freeimage as an independent library :-)

Well, maybe we can leave these source files for Windows, but definitely not for Mac and Linux.

**Comment by almarklein** [comment]
When can we `conda install freeimage`? When we do, I can test/make sure that imageio can find the lib. Then do a new release and update this recipe...

**Comment by ccordoba12** [comment]
It depends on Ilan, but right now you can use the recipe mentioned by @blink1073 above to build a conda package for `freeimage` on Linux/Mac and then add it as a dependency of `imageio`.

That way you can test if conda-build works correctly in this case (although i don't expect you to have big problems with that :-).

**Comment by almarklein** [comment]
Ah yes, I can also build it myself :P (It must be getting late...)

**Comment by almarklein** [comment]
Closing this in favour of a new PR to come.

**Pull request closed without merging**